### DISCUSSION TITLE
Logic Apps HTTP request actions omit headers without warning

### DISCUSSION THREAD

**Discussion by ViggoL** [Azure/logicapps#689] [Feature Ideas/Requests]
### Describe the Bug

Although this issue is described (far down) in the documentation, it's not acceptable for an HTTP action to simply omit headers, without warnings, before making a request. The headers omitted by default are also important ones that are expected to be included.

Either don't omit these headers, or inform the user in the designer somehow. We spent multiple hours debugging this before coming to the conclusion that the workflow does not behave as defined.

### Plan Type

Standard

### Steps to Reproduce the Bug or Issue

1. Create an HTTP action with Content-Type header and method = DELETE
2. Run the workflow

AB#16556949

**Comment by rohithah** [comment]
I moved this under discussion since this is more of change request. The current need for removing some of the headers come from the fact that Logic Apps provides several abstractions on top of basic Http. For example, you can specify how authenticate with the remote service without really having to populate auth headers and that requires us not allowing users to set the authorization header.

is there any particular header that you need be able to use?
